openapi-v1.yaml

```diff
@@ -3840,6 +3905,81 @@ paths:
               message: "Error try again"
               code: 1
 
+  /arrays/groups/{namespace}:
```

Comment:
We should not have a "group" per asset. This design needs to be revised. Let's talk today on this.